File: output/qwen2.5-coder-32b-instruct-fp16/code_snippet_34.cs
using System;

class Program {
    static long Factorial(int n) {
        if (n == 0) return 1;
        long result = 1;
        for (int i = 1; i <= n; i++) {
            result *= i;
        }
        return result;
    }

    static void Main() {
        Console.WriteLine(Factorial(5)); // Output: 120
    }
}

File: output/qwen3-30b-a3b-fp16/code_snippet_34.cpp
#include <iostream>
using namespace std;

int Factorial(int n) {
    if (n == 0)
        return 1;
    int result = 1;
    for (int i = 1; i <= n; i++)
        result *= i;
    return result;
}

// Example usage
int main() {
    cout << Factorial(5) << endl;  // Output: 120
    return 0;
}